crates/client/rpc/src/lib.rs

```diff
@@ -977,6 +977,53 @@ where
         Ok(estimates)
     }
 
+    /// Estimate the L2 fee of a message sent on L1
+    ///
+    /// # Arguments
+    ///
+    /// * `message` - starknet transaction request
+    /// * `block_id` - hash of the requested block, number (height), or tag
+    ///
+    /// # Returns
+    ///
+    /// * `fee_estimation` - the fee estimation (gas consumed, gas price, overall fee, unit)
```

Comment:
`fee_estimation` is not a variable that exists anywhere. Return objects are not named in rust.
I think you can just use the type itself here